src/components/CommentContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+
+const CommentContainer = ({ article_id }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetch(
+      `https://project1-be-nc-news.onrender.com/api/articles/${article_id}/comments`
+    )
+      .then((response) => response.json())
+      .then((response) => {
+        console.log(response);
+        setComments(response.comments);
+      })
+      .catch(() => {
+        setIsError(true);
+        setIsLoading(false);
+      });
+  }, [article_id]);
```

Comment:
It is unlikely that the article_id would change without navigating to a new page, this array needs to exist but this doesn't need to be inside it.